app/views/declaration/goods_location.scala.html

```diff
@@ -54,7 +54,7 @@
         @pageTitle(messages("declaration.goodsLocation.title"))
 
         @exportsInputText(
-        field = form("code"),
+        field = form("dev-goods-location-code"),
```

Comment:
why this change?

Reply:
This won't work, because this filed must be the same like form field

Reply:
I was trying to make the the following element consistent with the prototype's:
<input class="govuk-input" id="dev-local-reference-number" name="dev-local-reference-number" type="text" aria-describedby="dev-local-reference-number-hint">

Anu informed me that the tests were failing when the previous elements were listed as:
<input class="govuk-input govuk-!-width-two-thirds" id="code" name="code" type="text" aria-describedby="code-hint">